ui/selectors/selectors.js

```diff
@@ -1441,7 +1443,11 @@ export function getIsTransactionSecurityCheckEnabled(state) {
  * @returns Boolean
  */
 export function getIsSecurityAlertsEnabled(state) {
-  return state.metamask.securityAlertsEnabled;
+  // if the old settings is on, then return true else return false
+  return (
+    state.metamask.securityAlertsEnabled ||
+    state.metamask.transactionSecurityCheckEnabled
```

Comment:
Why are we checking the old preference (`transactionSecurityCheckEnabled`)? Isn't that for the old security check `opensea + blockaid`?

Reply:
There's a requirement to enable blockaid if the user has the old security check enabled.

Reply:
This should be achieved with migrations and not by checking both properties... otherwise we will need to support the two options until a migration is done.

Reply:
Yeah, doing the migration script now.